mappings/net/minecraft/block/HoneyBlock.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/unmapped/C_pxqujrqs net/minecraft/block/HoneyBlock
+	FIELD f_celsjlky START_SLIDE_WHEN_FALLING_SPEED_LEAST D
 	FIELD f_dgvsqrnu SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_hpjvfult MIN_FALL_SPEED_TO_BE_SLIDING D
+	FIELD f_htibcyrg THROTTLE_SLIDE_SPEED_TO D
```

Comment:
This doesnt feel like a good name

Reply:
The TO at the end is weird

Reply:
Personally i think this one makes a lot of sense, so I'll wait before changing it

Reply:
I'm not a fan of the `TO` at the end either

Reply:
I agree with the `TO` but the rest of it i think is fine

Reply:
Yeah, I'd remove the `TO`